.github/pull_request_template.md

```diff
@@ -7,17 +7,10 @@ Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.
 * Detailed list or prose of changes
 * ...
 
-## Test Plan
-Things that should succeed before merging.
+## PR Checklist
 
+- [ ] Update CHANGELOG
 - [ ] Unit tests
 - [ ] Adhoc testing
-- [ ] Update CHANGELOG
-- [ ] Run `./bin/eslint-ratchet` and verify that eslint errors have not increased
```

Comment:
Should we remove `eslint-ratchet` and `.eslint-ratchet-high-water-mark` if we're not using them anymore?

Reply:
They are removed, that was part of the change.